### ISSUE TITLE
Plexamp Support 

### ISSUE THREAD

**Issue by UndyingSoul** [Ottercast/OtterCastAudioV2#15]
Recently saw that Plexamp can be installed as a node.js server on a Raspberry Pi. Wondering if it would be possible on an Ottercast, this would be incredibly helpful, and would likely bring in a bunch of Plex fans, given the possibility of multi-room support in Plexamp.

Thoughts?
https://github.com/odinb/bash-plexamp-installer

**Comment by Manawyrm** [comment]
Hm, unsure.

Running node.js software is probably beyond both the RAM and CPU of the S3 SoC. 
There seems to be a armv7 compiled version of the plexamp binary, so it might be possible to try and run the binary, but I have no idea how well it would run.
I wouldn't get my hopes up. 

Additional problems:
- Binaries are propritary, we can't ship/include them
- Plex subscription is required. I do have a Plex Media Server setup, but no subscription.

**Comment by UndyingSoul** [comment]
Makes sense. How would one go about loading the binary to the ottercast? Once I get all the parts to build my own ottercast, I can write a simple document on how to install it. I've got a plex pass, so I can test.

**Comment by Manawyrm** [comment]
You could probably try to just "scp" the [Plexamp-Linux-headless-v4.6.1.tar.bz2](https://plexamp.plex.tv/headless/Plexamp-Linux-headless-v4.6.1.tar.bz2) file (more specifically it's contents) to the root FS of the Ottercast and try to run it.
I suspect that you'll need an external node.js binary as well. Either you can find a statically compiled node.js armv7 somewhere or you'll have to setup the buildroot setup, change the configuration to include buildroot and then build a new image from scratch.

**Comment by UndyingSoul** [comment]
Sweet, once I have time to order and build the Ottercast, I'll try it. Thanks for the help. I'll keep you posted.
